src/sentry/api/endpoints/organization_metrics_tags.py

```diff
@@ -52,6 +53,14 @@ def get(self, request: Request, organization: Organization) -> Response:
         if not is_mri(metric_name):
             raise BadRequest(message="Please provide a valid MRI to query a metric's tags.")
 
+        if all(
+            features.has("projects:use-eap-spans-for-metrics-explorer", project)
+            for project in projects
+        ):
+            if metric_name.startswith("d:eap"):
+                # TODO hack for EAP, return a fixed list
+                return Response([Tag(key="color"), Tag(key="location")])
```

Comment:
Similar to the other comment, could this not still go to the metrics platform for now?

Reply:
sure, but this PR is already pretty big and I'd like to add that logic incrementally